### ISSUE TITLE
problem running SPAdes error -11

### ISSUE THREAD

**Issue by tmms345** [ablab/spades#1191]
### Description of bug

Hi, 
When I try to run my assembly it gives me this error:

== Error == system call for: "['/home/name/salmo/SPAdes-3.15.4-Linux/bin/spades-hammer', '/home/name/salmo/Spades_out_dir/ISZ-2_S5/corrected/configs/[config.info](http://config.info/)']" finished abnormally, OS return value: -11

I tried running on both linux and WSL, but with the same result and I tried also with conda.

spades2.log is from linux
spades1.log in from WSL

params.txt is from linux

### spades.log

[spades2.log](https://github.com/ablab/spades/files/12806756/spades2.log)
[spades1.log](https://github.com/ablab/spades/files/12806762/spades1.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/12806806/params.txt)

### SPAdes version

SPAdes-3.15.4-Linux

### Operating System

Ubuntu 22.04.3 LTS

### Python Version

Python 3.9.12

### Method of SPAdes installation

Binaries, Conda

### No errors reported in spades.log

- [X] Yes

**Comment by tmms345** [comment]
How I can check it, to be sure?
Probably 2 GB

Il giorno mer 4 ott 2023 alle 20:24 Anton Korobeynikov <
***@***.***> ha scritto:

**Comment by robertwhbaldwin** [comment]
everyone is posting these errors to the board. 
You need to compile spades your self using spades_compile.py and it will work fine.

**Comment by asl** [comment]
> How I can check it, to be sure? Probably 2 GB

I doubt that this is enough amount for anything non-trivial

**Comment by tmms345** [comment]
Sorry I don’t understand, I have to run this command?

Il giorno mer 4 ott 2023 alle 22:03 robertwhbaldwin <
***@***.***> ha scritto:

**Comment by tmms345** [comment]
Yes, but I tried with another pc, 8Gb RAM, and I obtained the same result

Il giorno mer 4 ott 2023 alle 22:08 Anton Korobeynikov <
***@***.***> ha scritto:

**Comment by robertwhbaldwin** [comment]
How are you installing spades on your computer? Are you using the pre-compiled binaries or are you compiling it your self ?

**Comment by tmms345** [comment]
What do you mean by pre-compiled?


Il giorno mer 4 ott 2023 alle 22:38 robertwhbaldwin <
***@***.***> ha scritto:

**Comment by tmms345** [comment]
I think Pre-compiled, because I copied and posted on my bash

Il giorno mer 4 ott 2023 alle ore 22:38 robertwhbaldwin <
***@***.***> ha scritto:

**Comment by Arthurdyu** [comment]
the same problem！

**Comment by Arthurdyu** [comment]
Have you solved this error?  really appreciate ！
